src/main/java/org/broadinstitute/hellbender/utils/read/mergealignment/SamHeaderUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.hellbender.utils.read.mergealignment;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMTextHeaderCodec;
+import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.util.StringLineReader;
+
+public class SamHeaderUtils {
```

Comment:
do we really need a separate utils class for this? can you merge with ReadUtils?